main/relational_to_grid.es6.js

```diff
@@ -0,0 +1,115 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+const path = require('path');
+const process = require('process');
+const url = require('url');
+
+require('foam2');
+
+require('../lib/grid_dao.es6.js');
+require('../lib/http_json_dao.es6.js');
+require('../lib/json_dao_container.es6.js');
+require('../lib/local_json_dao.es6.js');
+require('../lib/web_apis/release.es6.js');
+require('../lib/web_apis/release_interface_relationship.es6.js');
+require('../lib/web_apis/web_interface.es6.js');
+const pkg = org.chromium.apis.web;
+
+const logger = foam.log.ConsoleLogger.create();
+
+const USAGE = `USAGE:
+
+    node /path/to/import.es6.js BaseConfluenceDataURL
+
+        BaseConfluenceDataURL = absolute https: or file: URL to directory
```

Comment:
This script could use some documentation about what it does.

Reply:
Done (push pending).